### DISCUSSION TITLE
Qt client: ability to get a tt file of all of your servers at once like teamtalk android.

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1711] [Ideas]
### Description

in android client, if you have not joined any server yet, if you press the more option button at the top of the screen, there will be an export server list option, if you klick on it, you will be then just asked for the path where you want the tt file to be stored, you will be then get a .tt file contanes all your servers: this feecher will be really useful for the Qt client for getting backup of your servers just in kase you reinstall windows, you reset your disk, you change your computer, and more: only a .tt file contanes all your servers is really bettar than havving 100 .tt files for getting backups: my idea is add a new button called export server list to the connect to a server dialogue, if klicked on it, ask for the path, and once the path is selected, generate a .tt file contanes all your servers  in that path: and anytime if we want to get our server list, we use the inport tt file option wich is already exist.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
and if the server list is completely empty and has no server on it, the xport server list option should be disabled

**Comment by amirmahdifard** [comment]
seriously, what is rong with my ideas that you are always doing this? please tell me let me know!

**Reply by bear101** [reply]
Post your ideas in "Discussions" under the "Ideas" category:
https://github.com/BearWare/TeamTalk5/discussions/categories/ideas

If others find your ideas useful then, maybe, it will become an issue one day.

**Comment by amirmahdifard** [comment]
@bear101 ok but people are not giving comments
